src/AElf.Contracts.TestKit/MockTxHub.cs

```diff
@@ -39,7 +40,7 @@ public async Task AddTransactionsAsync(IEnumerable<Transaction> transactions)
             var txs = transactions.ToList();
             foreach (var transaction in txs)
             {
-                _allTransactions.Add(transaction.GetHash(), transaction);
+                _allTransactions.TryAdd(transaction.GetHash(), transaction);
```

Comment:
if(TryAdd){
}else{
throw new OperationException("");
}

if we have any failed Add Operation, we should know it

Reply:
@jason-hoopox And alos check all Try Operation in aelf codes